lib/views/login_signup.dart

```diff
@@ -0,0 +1,539 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class login_signup extends StatefulWidget{
+  login_signup_state createState() => login_signup_state();
+}
+
+class login_signup_state extends State<login_signup>{
+
+  TextEditingController _txt1 = TextEditingController();
+  TextEditingController _txt2 = TextEditingController();
+  TextEditingController _username = TextEditingController();
+  TextEditingController _password = TextEditingController();
+  TextEditingController _email = TextEditingController();
+  TextEditingController _contact = TextEditingController();
+  FocusNode fnode1 = FocusNode();
+  FocusNode fnode2 = FocusNode();
+  FocusNode fnode3 = FocusNode();
+  FocusNode uname = FocusNode();
+  FocusNode password = FocusNode();
+  FocusNode email = FocusNode();
+  FocusNode contact = FocusNode();
+  final _formKey = GlobalKey<FormState>();
+  final _formKey2 = GlobalKey<FormState>();
+  late double h1,h2,h3; // this will be used for setting margin over waves
```

Comment:
This is not a good practice to name a variable. Variable names should always be meaningful.